### PR TITLE
[Integration Test][omevv_firmware]: Negative Scenarios

### PR DESCRIPTION
**Pull request by Kritika-Bhateja-03**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: tests/integration/targets/omevv_firmware/tests/invalid_module_specific.yaml [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+            firmware_components: "DCIM:INSTALLED#802__Diagnostics.Embedded.1:LC.Embedded.1"
+      register: partial_cluster_result
+      ignore_errors: true
+
+    - name: Verify task status - Negative - Update the firmware by passing passing partial cluster_name.
+      ansible.builtin.assert:
+        that:
+          - partial_cluster_result.skipped
+          - not partial_cluster_result.changed
+          - partial_cluster_result.msg
```
Comment (ABHISHEK-SINHA10) [inline tests/integration/targets/omevv_firmware/tests/invalid_module_specific.yaml]: assert the msg

File: tests/integration/targets/omevv_firmware/tests/invalid_module_specific.yaml [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+            firmware_components: "DCIM:INSTALLED#802__Diagnostics.Embedded.1:LC.Embedded.1"
+      register: invalid_job_wait_timeout_type
+      ignore_errors: true
+
+    - name: Verify task status - Negative - invalid job_wait_timeout type
+      ansible.builtin.assert:
+        that:
+          - invalid_job_wait_timeout_type.failed
+          - not invalid_job_wait_timeout_type.changed
+          - '"argument ''job_wait_timeout'' is of
```
Comment (ABHISHEK-SINHA10) [inline tests/integration/targets/omevv_firmware/tests/invalid_module_specific.yaml:220]: I think this msg is coming from python, we should handle gracefully, like "Given job_wait_timeout is not int type." check with team and raise defect

File: tests/integration/targets/omevv_firmware/tests/invalid_module_specific.yaml (hunk 1 above)
Comment (Kritika-Bhateja-03) [reply tests/integration/targets/omevv_firmware/tests/invalid_module_specific.yaml]: done

File: tests/integration/targets/omevv_firmware/tests/invalid_module_specific.yaml (hunk 2 above)
Comment (Kritika-Bhateja-03) [reply tests/integration/targets/omevv_firmware/tests/invalid_module_specific.yaml:220]: This is coming from ansible directly. Lets keep it same,.

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Review by Saksham-Nautiyal — APPROVED** [review]

**Pull request merged**